var/spack/repos/builtin/packages/py-warpx/package.py

```diff
@@ -50,7 +50,8 @@ class PyWarpx(PythonPackage):
     depends_on('py-numpy@1.15.0:1', type=('build', 'run'))
     depends_on('py-mpi4py@2.0.0:', type=('build', 'run'), when='+mpi')
     depends_on('py-periodictable@1.5:1', type=('build', 'run'))
-    depends_on('py-picmistandard@0.0.14', type=('build', 'run'))
+    depends_on('py-picmistandard@0.0.14', type=('build', 'run'), when='@21.03:21.11')
+    depends_on('py-picmistandard@0.0.16', type=('build', 'run'), when='@21.12')
     depends_on('py-setuptools@38.6:', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools@42:', type='build')
    depends_on('py-cmake@3.15:3', type='build')
```
From the `pyproject.toml`